qutebrowser/browser/commands.py

```diff
@@ -900,6 +900,8 @@ def tab_move(self, direction: {'type': ('+', '-')}=None, count=None):
                 new_idx = self._current_index() - delta
             elif direction == '+':  # pragma: no branch
                 new_idx = self._current_index() + delta
+            if config.get('tabs', 'wrap'):
+                new_idx = new_idx % self._count()
```

Comment:
This could be `new_idx %= self._count()` instead, but I'll just fix this after merging as it's a minor style thing.